src/jsModal.php

```diff
@@ -7,12 +7,61 @@
  */
 class jsModal extends jsExpression
 {
-    public function __construct($title, $url, $args = [], $mode = 'json')
+    public function __construct($title, $url, $args = [], $mode = 'json', $hasHeader = true)
```

Comment:
Why don't we use seed mechanism and init() method?

Reply:
no need for extra arg, setting first argument to false/null/empty string would be a intuitive way to switch off header.